update-server/tests/integration/features/update.beta.feature

```diff
@@ -42,7 +42,7 @@ Feature: Testing the update scenario of releases on the beta channel
     And The received version is "9.0.10"
     When The request is sent
     Then The response is non-empty
-    And URL to download is "https://download.owncloud.org/community/owncloud-9.1.6.zip"
+    And URL to download is "https://download.owncloud.org/community/owncloud-9.0.11RC2.zip"
```

Comment:
9.1.7RC2, we serve the next major update ?

Reply:
If it is the latest minor version, should be don't do it?